{"timestamp":1727826060194,"message":"Database error","log.level":"ERROR","logger.name":"api.blueprints.drbLink","thread.id":139916312742784,"thread.name":"MainThread","process.id":2303,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/api/blueprints/drbLink.py","line.number":36,"entity.type":"SERVICE"}
